src/main/java/net/atos/zac/policy/PolicyService.java

```diff
@@ -95,6 +95,7 @@ public ZaakRechten readZaakRechten(final Zaak zaak, final ZaakType zaaktype) {
         return readZaakRechten(zaak, zaaktype, null);
     }
 
+    @Deprecated
```

Comment:
Always good to add a deprecation reason

Reply:
Not in Java as it seems :(

Reply:
Extracted this in https://dimpact.atlassian.net/browse/PZ-4443